build/xtask/src/dist.rs

```diff
@@ -573,6 +575,66 @@ fn write_gdb_script(cfg: &PackageConfig, image_name: &str) -> Result<()> {
     Ok(())
 }
 
+/// Encode the given data as a tagged TLV-C chunk
+fn data_to_tlvc(tag: &str, data: &[u8]) -> Result<Vec<u8>> {
+    if tag.len() != 4 {
+        bail!("Tag must be a 4-byte value, not '{}'", tag);
+    }
+    let mut out = vec![];
+    let mut header = tlvc::ChunkHeader {
+        tag: tag.bytes().collect::<Vec<u8>>().try_into().unwrap(),
+        len: 0.into(),
+        header_checksum: 0.into(),
+    };
+    out.extend(header.as_bytes());
+    let c = tlvc::begin_body_crc();
+    let mut c = c.digest();
+    c.update(data);
+    out.extend(data);
+    let body_len = out.len() - std::mem::size_of::<tlvc::ChunkHeader>();
+    let body_len = u32::try_from(body_len).unwrap();
+    while out.len() & 0b11 != 0 {
```

Comment:
Maybe worth a comment that tlvc requires the body to be padded to a multiple of four (which is not included in the length)?

Nit: I think this would be marginally clearer as `while out.len() % 4 != 0`, but I could be in the minority.

Reply:
Added a comment!  (This is copy-pasta from the TLV-C `texttool` example, so I'll leave it be otherwise)